styleguide/_themes/global/scss/_buttons.scss

```diff
@@ -1,5 +1,5 @@
 /* the button class can be used for both anchor and input tags, defaults to medium/gray */
-.button, a.button, button{
+.button, button{
```

Comment:
I see the CTA examples on `_solutions.html` floating oddly:

![screen shot 2016-04-20 at 3 43 25 pm](https://cloud.githubusercontent.com/assets/1407330/14689832/9f20e33c-070e-11e6-87a8-5f02dcb7d583.png)


Need to add `display: block` to this.